dev/devicelab/bin/tasks/plugin_test_ios.dart

```diff
@@ -10,6 +10,7 @@ import 'package:flutter_devicelab/framework/framework.dart';
 Future<Null> main() async {
   await task(combine(<TaskFunction>[
     new PluginTest('ios', <String>['-i', 'objc']),
-    new PluginTest('ios', <String>['-i', 'swift']),
+    // TODO(mravn): Re-enable this once CocoaPods is upgraded on device lab.
+    // new PluginTest('ios', <String>['-i', 'swift']),
```

Comment:
Alternatively you could mark the test as flaky until it becomes green _with_ Swift support.

Reply:
Right. But it isn't flaky. It'll fail consistently...

Reply:
We also use the flaky flag when debugging new tests or big changes to tests when we're not sure that they will pass. If they look green after a few commits, we flip the flag.

Reply:
OK. Test marked flaky instead.